interface/ceed-preconditioning.c

```diff
@@ -885,7 +885,23 @@ static int CeedSingleOperatorMultigridLevel(CeedOperator op_fine, CeedVector p_m
   CeedCheck(!is_composite, ceed, CEED_ERROR_UNSUPPORTED, "Automatic multigrid setup for composite operators not supported");
 
   // Coarse Grid
-  CeedCall(CeedOperatorCreate(ceed, op_fine->qf, op_fine->dqf, op_fine->dqfT, op_coarse));
+  {
+    bool is_at_points;
+
+    CeedCall(CeedOperatorIsAtPoints(op_fine, &is_at_points));
+    if (is_at_points) {
+      CeedVector          point_coords;
+      CeedElemRestriction rstr_points;
+
+      CeedCall(CeedOperatorCreateAtPoints(ceed, op_fine->qf, op_fine->dqf, op_fine->dqfT, op_coarse));
```

Comment:
Note: Here only the fine and coarse grid operators are AtPoints. There's no benefit to making the restriction and prolongation AtPoints that I can see since we're simply transferring between FEM grids.

Reply:
I agree.